test/models/ability_test.rb

```diff
@@ -633,8 +611,8 @@ def teardown
     m = create_valid_media
     pma = create_project_media project: pa, media: m
     pmb = create_project_media project: pb, media: m
-    c1 = create_comment annotated: pma
-    c2 = create_comment annotated: pmb
+    task1 = create_task annotated: pma
+    task2 = create_task annotated: pmb
```

Comment:
Why do you need to replace `create_comment` with `create_task`? I think we should only remove `create_comment`